src/content/docs/guides/tour.md

```diff
@@ -135,9 +135,7 @@ length(?X, STRLEN(?X)) :- input(?X) .
 
 Only one `length` fact will be computed, namely `length("example", 7)`, but there will not be any visible error.
 
-In general, many functions in Nemo are named like corresponding functions in SPARQL, and Nemo strives to implement them in the same way.
-
-***Known restriction in Nemo v0.5.0.*** As of Nemo v0.5.0, built-in functions are strictly case-sensitive. Using functions that are not recognized by Nemo leads to an error (in fact, a panic). Future versions of Nemo will support user-defined nested functions.
+In general, many functions in Nemo are named like corresponding functions in SPARQL, and Nemo strives to implement them in the same way. Built-ins are not case-sensitive.
```

Comment:
If built-ins are not case sensitive, why is it most functions are written in ALL UPPERCASE yet checks like `isInteger` are written in camel case?